libcontainer/cgroups/fs2/io.go

```diff
@@ -30,7 +30,16 @@ func setIo(dirPath string, cgroup *configs.Cgroup) error {
 		filename := "io.bfq.weight"
 		if err := fscommon.WriteFile(dirPath, filename,
 			strconv.FormatUint(uint64(cgroup.Resources.BlkioWeight), 10)); err != nil {
-			return err
+			// if io.bfq.weight does not exist, then bfq module is not loaded.
+			// Fallback to use io.weight with a conversion scheme
+			if os.IsNotExist(err) {
```

Comment:
Wonder if it works in case openat2 is used; let me check...

Yes, it works. Good.